docs_samples/03_python_app_custom_metrics.md

```diff
@@ -0,0 +1,54 @@
+# Capture Custom Metrics
+
+Custom metrics are helpful for reporting on specific metric values from your application.  These are typically made up of the metric name and the value.  This data will then be placed into the `customerMetrics` table in Application Insights.
+
+There are two types of metrics that can be sent to Application Insights via the [TrackMetric API](https://docs.microsoft.com/en-us/azure/azure-monitor/app/api-custom-events-metrics#trackmetric):
+
+- Single Value - Each time the value is set to Application Inights.
+- Aggregation - An aggregation of the values from your application.  
+
+Each time a call is made to `trackMetric` a new row is generated in Application Insights.
+
+## Simple Application
+
+- Open the `./SimpleApps/metric.py` file
+- Press **F5** to run the file, select **Python file** in the debug configuration window
+- Switch to the Azure Portal
+- Browse to your lab resource group
+- Browse to the `python-appinsights-SUFFIX` application insights resource and select it
+- Under **Monitoring**, select **Metrics**
+- For the **Metric namespace**, select **carrots_view**
+- For the **Metric**, select **carrots_view**
+- You should see some data displayed:
+
+    ![The custom metric for the carrots view is displayed.](./media/python_custommetrics-carrots.png "Review the results of the metric data.")
```

Comment:
This file uses an example that already exists. I don't see much value in adding it again into samples.

Reply:
I don't mind referencing an existing one but I don't see it in the current repo?  Its included in these samples as part of the visual studio code environment so they can easily execute it rather than downloading the code snippets from docs.

Reply:
I found your reference.  I am thinking maybe I put in the setup script another git call that pulls your samples, does that sound fair.  I agree they should not be in two places.

Reply:
That could work. Or simply only include unique samples (ones that are a bit more complex and require setup) in this repo, and then include at the bottom "for more examples see..." and then link to the main opencensus repo.

Reply:
Ok, I added a git clone from the opencensus repo and copy the samples into the simpleapps folder.